mscore/startcenter.cpp

```diff
@@ -124,8 +124,8 @@ void Startcenter::loadScore(QString s)
             newScore();
             }
       else {
-            mscore->openScore(s);
```

Comment:
Not sure about this change, what is the intent here?

Reply:
I thought, the start center would be seen as a modal dialog, so if it is still open when openScore is called, the tab won't be switched, while actually it should be. So, first close the start center, then open the score, and the tab will be switched.